lib/adal.js

```diff
@@ -367,7 +401,8 @@ AuthenticationContext.prototype.acquireToken = function (resource, callback) {
 
     // refresh attept with iframe
     //Already renewing for this resource, callback when we get the token.
-    if (this._activeRenewals[resource]) {
+    if (this._activeRenewals[resource] ||
+        (window.parent && window.parent.AuthenticationContext()._activeRenewals[resource])) {
```

Comment:
Why did you add the additional checks? The renewal operation always happens in an iframe, I dont think we need to check the window.parent property here.